<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Port Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="port_name_Comb">
           <item>
            <property name="text">
             <string>com1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>com10</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Data Bits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="data_bits_Comb">
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Baud Rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baud_rate_Comb">
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Stop Bits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stop_bits_Comb">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Parity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="parity_Comb">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Odd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Even</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MARK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SPACE</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Timeout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timeout_spinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QRadioButton" name="is_Open_Btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="open_port_Btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_port_Btn">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="horizontalWidget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Info Panel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="info_Panel"/>
         </item>
         <item>
          <widget class="QRadioButton" name="save_data_Btn">
           <property name="text">
            <string>Save Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="screen_shot_Btn">
           <property name="text">
            <string>Print Screen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="send_Panel"/>
         </item>
         <item>
          <widget class="QPushButton" name="send_Btn">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="plot_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QwtPlot" name="qwtPlot"/>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <widget class="QSlider" name="y_pos_Scope">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="y_neg_Scope">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Number of Points to display                       </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="num_points_Slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Refresh Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="refresh_Time"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QRadioButton" name="en_filter_Btn">
           <property name="text">
            <string>Enable filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="filter_Combo">
           <item>
            <property name="text">
             <string>db4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>db5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>db6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sym4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sym5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sym6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>coif4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>coif5</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>           Level</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="leve_In"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
